src/ApprenticeAan/SFA.DAS.ApprenticeAan.Api.UnitTests/Controllers/CalendarEventsControllerTests/GetCalendarEventsQueryTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using SFA.DAS.ApprenticeAan.Api.Models;
+using SFA.DAS.ApprenticeAan.Application.CalendarEvents.Queries.GetCalendarEvents;
+using SFA.DAS.ApprenticeAan.Application.Common;
+
+namespace SFA.DAS.ApprenticeAan.Api.UnitTests.Controllers.CalendarEventsControllerTests;
+public class GetCalendarEventsRequestModelTests
+{
+    [Test, AutoData]
+    public void Operator_PopulatesModelFromParameters(Guid memberId, DateTime? fromDate, DateTime? toDate, List<EventFormat> eventFormats, List<int> calendarIds, List<int> regionIds, int? page, int? pageSize)
```

Comment:
Why are we testing operator in controller?

Reply:
I have it in the wrong folder, moving it.